crates/esplora/tests/async_ext.rs

```diff
@@ -186,12 +186,14 @@ pub async fn test_async_update_tx_graph_gap_limit() -> anyhow::Result<()> {
         sleep(Duration::from_millis(10))
     }
 
-    // A scan with a gap limit of 2 won't find the transaction, but a scan with a gap limit of 3
+    // A scan with a gap limit of 3 won't find the transaction, but a scan with a gap limit of 4
     // will.
-    let (graph_update, active_indices) = env.client.full_scan(keychains.clone(), 2, 1).await?;
+    // FIXME: this part of the test is erroring
+    let (graph_update, active_indices) = env.client.full_scan(keychains.clone(), 3, 1).await?;
     assert!(graph_update.full_txs().next().is_none());
     assert!(active_indices.is_empty());
-    let (graph_update, active_indices) = env.client.full_scan(keychains.clone(), 3, 1).await?;
+    // NOTE: here onwards is OK
+    let (graph_update, active_indices) = env.client.full_scan(keychains.clone(), 4, 1).await?;
```

Comment:
Anyone knows why this is happening?
It is failing in:

```rust
assert!(graph_update.full_txs().next().is_none());
assert!(active_indices.is_empty());
```

everything else works...